test/unit/storage/entities/peer.js

```diff
@@ -302,12 +302,16 @@ describe('Peer', () => {
 				executeFile: sinonSandbox.stub().resolves([validPeer]),
 				parseQueryComponent: sinonSandbox.stub(),
 			};
+
+			const localFields = [...validPeerFields].filter(
```

Comment:
I don't see the need of deconstructing here.

Reply:
Done. Resolving.